Packs/GRR/Scripts/GrrGetFiles/GrrGetFiles.yml

```
@@ -5,9 +5,10 @@ name: GrrGetFiles
 script: ''
 type: python
 subtype: python2
+deprecated: true
 tags:
 - GRR
-comment: Downloads files from specified machine without requiring approval
+comment: Deprecated. Use grr_get_files instead.
```

Relevant for all:
Do not refer to the deprecated command.
```suggestion
comment: Deprecated. Use grr-get-files instead.
```